explorer/package.json

```diff
@@ -32,23 +32,25 @@
   },
   "main": "src/index.ts",
   "dependencies": {
+    "@types/express-pino-logger": "^4.0.2",
```

Comment:
Can this be moved to "devDependencies"?

Reply:
You can do so via
```sh
yarn remove @types/express-pino-logger
yarn add -D @types/express-pino-logger
```

Reply:
No, then we would have no request and response logging :(

Reply:
This is just moving the type definition to dev dependencies. It's not used during runtime.